### ISSUE TITLE
[regression]Play Music, Play Books, Drive, Photos, and Maps cannot find Google account

### ISSUE THREAD

**Issue by oldosfan** [microg/android_packages_apps_GmsCore#810]
0.2.6 worked perfectly, but with 0.2.7 they all fail to discover the Google account and insist I log in. 

I can't find any errors  in the logcat, which might mame this harder to debug

**Comment by Nanolx** [comment]
Google Fotos doesn't need EXTENDED_ACCESS permission, I've just checked it.

**Comment by ale5000-git** [comment]
@oldosfan: If you have a problem with this app please post a logcat.

**Comment by oldosfan** [comment]
Sure :) Freshly baked logcat coming up!

**Comment by ale5000-git** [comment]
It may have been already fixed by 4713797edc21e3f3b2a26194056f63b307e67ef6 (which isn't included in the latest release).

I suggest to wait the new version or compile it yourself to test.

**Comment by oldosfan** [comment]
Sure, I'll set up a GitLab CI/CD instance to build hourlies and test

**Comment by Nanolx** [comment]
Given your changed description (which doesn't make sense, as it now is basically a completely different issue) this is now a duplicate of #790 and your sarcasm aside: it has been tested that the fixes not yet released have/will solve your issue.

**Comment by oldosfan** [comment]
Sure! Thanks for the tip; 

TL; DR it was really late last night and I had time to test hourlies. Apparently you were right; Thanks!

However, Play Console still fails to log in. Perhaps it needs EXTENDED_ACCESS?
